src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -117,3 +119,30 @@ Napi::Value ModelWrap::get_outputs(const Napi::CallbackInfo& info) {
 
     return js_outputs;
 }
+
+Napi::Value ModelWrap::set_friendly_name(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() != 1 || !info[0].IsString()) {
+        reportError(env , "Expected a single string argument for the friendly name" );
+        return env.Undefined();
+    }
+    const auto name = info[0].As<Napi::String>().Utf8Value();
+    try {
+        _model->set_friendly_name(name);
+    } catch (const std::exception& e) {
+        reportError(env , e.what());
+        return env.Undefined();
```

Comment:
You can remove this line, because you already have return below
```suggestion
        return env.Undefined();
```